src/domain/accessibilityStatement/accessibilityStatement.module.scss

```diff
@@ -3,6 +3,7 @@
 
 .accessibilityStatement {
   margin-bottom: $largeMargin;
+  max-width: $textParagraphMaxWidth;
```

Comment:
This look very duct tape

Reply:
Any idea on how to improve on this? The issue is that the lines should not be too long, it is difficult to read text which has very long lines.